K2Bridge/Visitors/Aggregations/MetricAggregationVisitor.cs

```diff
@@ -33,6 +34,29 @@ public void Visit(CardinalityAggregation cardinalityAggregation)
             cardinalityAggregation.KustoQL = $"{EncodeKustoField(cardinalityAggregation.Key)}={KustoQLOperators.DCount}({EncodeKustoField(cardinalityAggregation)})";
         }
 
+        /// <inheritdoc/>
+        public void Visit(ExtendedStatsAggregation extendedStatsAggregation)
+        {
+            Ensure.IsNotNull(extendedStatsAggregation, nameof(extendedStatsAggregation));
+            EnsureClause.StringIsNotNullOrEmpty(extendedStatsAggregation.Field, extendedStatsAggregation.Field, ExceptionMessage);
+
+            var sep = AggregationsConstants.MetadataSeparator;
+            var key = $"['{extendedStatsAggregation.Key}{sep}extended_stats']";
```

Comment:
Please use EncodeKustoField here too. Something similar to:

```suggestion
            var key = EncodeKustoField(extendedStatsAggregation.Key + AggregationsConstants.MetadataSeparator + "extended_stats");
```

Reply:
No, because EncodeKustoField will not return the expected value here

Reply:
Why? is there a decimal point there?

Reply:
Ah ! Indeed, we had a point previously, but we've changed it.
Good catch ! thx :)
Done